src/errorpage.cc

```diff
@@ -142,44 +142,49 @@ static void ValidateStaticError(const int page_id, const SBuf &inputLocation);
 
 /* local constant and vars */
 
-/**
- \ingroup ErrorPageInternal
- *
- \note  hard coded error messages are not appended with %S
- *      automagically to give you more control on the format
```

Comment:
This stale description was rather misleading because, in modern code, stored hard-coded error messages are never actually sent (so "control on the format" is useless) with the exception of ERR_SQUID_SIGNATURE, which is not a true error but the signature (i.e. `%S` substitution) itself (so "appending %S" would not make sense for it).